typescript/schemas/src/internal/types.ts

```diff
@@ -38,6 +39,7 @@ export type FoxgloveMessageSchema = {
   name: string;
   description: string;
   rosEquivalent?: keyof typeof import("@foxglove/rosmsg-msgs-common").ros1;
+  protoEquivalent?: "Timestamp" | "Duration";
```

Comment:
would be better for the docs if we referred to them by the fully qualified name like `google.protobuf.Timestamp`